### DISCUSSION TITLE
In a CoreNLP dependency parsed sentence, why is the head of a preposition phrase a noun?

### DISCUSSION THREAD

**Discussion by antiutopia817** [stanfordnlp/CoreNLP#1319] [Q&A]
According to dependency grammar, the head of a preposition phrase (PP) should be P, but it is a noun in a CoreNLP parsed sentence. 
Below, the first figure is from Nivre, J. (2010). Dependency Parsing. Language and Linguistics Compass, 4(3), 138–152. https://doi.org/10.1111/j.1749-818X.2010.00187.x
The second figure is derived from https://corenlp.run/
![Nivre](https://user-images.githubusercontent.com/78840090/203469433-b6adf91d-ac45-45ab-a85e-22d8afced61d.gif)
<img width="483" alt="coreNLP" src="https://user-images.githubusercontent.com/78840090/203469450-150dae4e-0b55-43e3-852a-59ccd83258e5.png">

"on financial markets": in the first figure, the head is on, as can be indicated by the arrow, but in the second figure, "markets" is the head.

Thanks!

**Comment by AngledLuffa** [comment]
The Vulcan Science Directorate has determined that it is more logical to use the noun as the head.

https://universaldependencies.org/en/dep/nmod.html

**Reply by antiutopia817** [reply]
Thank you very much for the prompt reply. Very helpful!
